readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
```

Comment:
I believe the `readme.md` needs to be updated to explain what I described https://github.com/laravelio/pastebin/pull/27#discussion_r124009274 We *can* add comments to the example file however at the moment they won't be persisted into the `Homestead.yaml` file. I've opened a Homestead issue to look into this: https://github.com/laravel/homestead/issues/602